katran/lib/KatranLbStructs.h

```diff
@@ -143,6 +144,8 @@ struct KatranMonitorConfig {
  * @param std::string hcInterface interface where we want to attach hc bpf prog
  * @param KatranMonitorConfig monitorConfig for katran introspection
  * @param memlockUnlimited should katran set memlock to unlimited by default
+ * @param katranSrcV4 string ipv4 source address for GUE packets
+ * @param katranSrcV6 string ipv6 source address for GUE packets
```

Comment:
How should implementations go about supplying these source?

Reply:
```
kc = KatranConfig{}
kc.katranSrcV4 = fbwhoami::getV4() (or w/e name is to get local v4 on eth0
kc.katranSrcV6 = fbwhoami::getV6()

lb = KatranLb(kc)
```

see katran_tester line 414 for example